### ISSUE TITLE
2.4.7 Empty headline is still output in the template

### ISSUE THREAD

**Issue by Ainschy** [madeyourday/contao-rocksolid-custom-elements#166]
Contao 4.13.37  PHP8.2

Since 2.4.7 empty headlines are displayed in the frontend despite a query in the template.

![2024-02-27_12h24_53](https://github.com/madeyourday/contao-rocksolid-custom-elements/assets/1788135/c6899bbe-0416-47ad-9dd8-5b8b7d36c973)
![2024-02-27_12h23_27](https://github.com/madeyourday/contao-rocksolid-custom-elements/assets/1788135/2d5e4490-6f78-49a4-8026-d2f82e1e26bb)

**Comment by zoglo** [comment]
I was debugging this for the past few minutes and this is because $this->headline is now of type Stringable and *not* empty in if-clauses anymore.

![image](https://github.com/madeyourday/contao-rocksolid-custom-elements/assets/55794780/655fa0d0-713d-4cd8-a76f-512a14d7a0a3)

This is being caused in the last change:
https://github.com/madeyourday/contao-rocksolid-custom-elements/blob/c69bf399c9fc6d87ab6114194a0a0ec16bf33466/src/Element/CustomElement.php#L301-L315

The question is if this should be fixed from your side @ausi or if every person using custom-elements would need to update their templates after updating to version 2.4.7.

We'd all either need to use ``<?php if ((string) $this->headline): ?>`` or there should be a workaround.

I consider this critical since updates would lead to something that may output empty headlines like here:

```
<div class="ce_rsce_text col-m-6 text block">
	<div class="inside">
		<div class="c_headline">
			<h1></h1>
		</div>
		<div class="c_text">…</div>
	</div>
</div>
```

**Comment by ausi** [comment]
@Ainschy can you please try if the latest `dev-master` version fixes your issue?

**Comment by Ainschy** [comment]
Thanks it's look fine for me

**Comment by ausi** [comment]
Released as version 2.4.8

@Ainschy @zoglo thank you for the quick reporting/debugging/testing!
